astroid/bases.py

```diff
@@ -120,7 +122,11 @@ def infer(self, context=None):
         yield self
 
 
-def _infer_stmts(stmts, context, frame=None):
+def _infer_stmts(
+    stmts: Sequence[nodes.NodeNG | type[Uninferable] | Instance],
+    context: InferenceContext | None,
+    frame: nodes.NodeNG | Instance | None = None,
```

Comment:
I haven't checked that again, but it seems to be fine, at least for now. We can come back to it  later if we need to.